sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -109,6 +109,34 @@ def test_rfe():
     assert_array_almost_equal(X_r, X_r_sparse.toarray())
 
 
+def test_rfe_negative_n_features():
+    clf = SVC(kernel="linear")
+    with pytest.raises(ValueError, match=r"n_features_to_select must be *"):
+        RFE(estimator=clf, n_features_to_select=-1, step=0.1).fit(...)
```

Comment:
My `fit(...)` suggestion means that `...` should be replaced with something meaningful. In this case:

```py
iris = load_iris()
rfe = RFE(estimator=clf, n_features_to_select=-1, step=0.1)
with pytest.raises(ValueError, match=r"n_features_to_select must be *"):
    rfe.fit(iris.data, iris.target)
```

Reply:
Ah, yes of course, sorry, I should have seen that!